kustomize/overlays/prod/shard/patches/deploy-config.yaml

```diff
@@ -4,10 +4,11 @@ data:
   EVENT_BRIDGE_INGRESS_IMAGE: quay.io/5733d9e2be6485d52ffa08870cabdee0/ingress:7afa309b717285d5454846ad8863af3d80a13403-jvm
   EVENT_BRIDGE_K8S_PLATFORM: openshift
   EVENT_BRIDGE_MANAGER_URL: http://event-bridge:8080
+  EVENT_BRIDGE_SSO_URL: https://keycloak-event-bridge-prod.apps.openbridge-dev.fdvn.p1.openshiftapps.com/auth/realms/event-bridge-fm
   EVENT_BRIDGE_SSO_CLIENT_ID: event-bridge-fm
   EVENT_BRIDGE_SSO_SECRET: secret
-  EVENT_BRIDGE_SSO_URL: https://keycloak-event-bridge-prod.apps.openbridge-dev.fdvn.p1.openshiftapps.com/auth/realms/event-bridge-fm/protocol/openid-connect/token
```

Comment:
yes, once I implemented the whole thing, now I know what values should the property have (110 and 111 should not have been splitted in the first place)